app/models/mrss_profile.rb

```diff
@@ -17,7 +17,7 @@ def initialize(options)
     super(options)
   end
 
-  def self.find_by_name(mrss_name)
+  def self.find_by_mrss_profile_name(mrss_name)
```

Comment:
Renamed to prevent our custom `find_by_name` from being confused (or silently conflicting) with Rails' built-in `find_by_*` methods. This was flagged by Rubocop.